src/layer/vector/Polygon.js

```diff
@@ -84,6 +84,34 @@ L.Polygon = L.Polyline.extend({
 		return this._map.layerPointToLatLng(center);
 	},
 
+	// @method contains (latlng: LatLng): Boolean
+	// Returns `true` if `latlng` is inside the (multi)polygon.
```

Comment:
Should the documentation say that the polyline has to be on a map in order for this method to work?